extensions/amp-recaptcha-input/validator-amp-recaptcha-input.protoascii

```diff
@@ -13,3 +13,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the license.
 #
+tags: {  # amp-recaptcha-input
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-recaptcha-input"
+    version: "0.1"
+    version: "latest"
+    deprecated_allow_duplicates: true
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-form> <amp-recaptcha-input>
+  html_format: AMP
+  tag_name: "AMP-RECAPTCHA-INPUT"
+  spec_name: "amp-form amp-recaptcha-input"
```

Comment:
I'm not sure this one really needs a spec_name. spec_name is typically optional, and usually used if there is more than one tag spec for the same tag name to help distinguish the rules. In your case, you only have one tag name, so I'd just drop the spec name.

It'll make the error messages a little clearer. Instead of something like "The specified layout 'RESPONSIVE' is not supported by tag 'amp-form amp-recaptcha-input'." it'll read "The specified layout 'RESPONSIVE' is not supported by tag 'amp-recaptcha-input'."